src/ui/popup.js

```diff
@@ -435,30 +437,36 @@ export default class Popup extends Evented {
      * Adds a CSS class to the popup container element.
      *
      * @param {string} className Non-empty string with CSS class name to add to popup container
+     * @returns {Popup} `this`
```

Comment:
Added a return to allow chaining

Reply:
Nit: During the doc's sprint, we updated all returns of `this` to be:

`@returns {...} Returns itself to allow for method chaining.`